### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by tatuiot** [thingsboard/thingsboard#9499]
**Describe the bug**
Upgrading from Thingsboard CE 3.4.4 to Thingsboard CE 3.5 fails when running the  upgrade script

**Your Server Environment**

* own setup
  * local infrastructure 
  * ThingsBoard Version 3.4.4
  * OS Name and Version Ubuntu Linux 20.04.6
  * AMD Ryzen 5 5600G with Radeon Graphics, 12 Cores
  * 32GB Ram
  * 1.4TB Disk space

**Your Client Environment**

**Desktop (please complete the following information):**

* OS: Ubuntu Desktop 22.04
* Browser Chrome 
* Version Versión 117.0.5938.149 (Build oficial) (64 bits)

**To Reproduce**
running the following commands from CLI
wget https://github.com/thingsboard/thingsboard/releases/download/v3.5/thingsboard-3.5.deb
sudo systemctl stop thingsboard
sudo dpkg -i thingsboard-3.5.deb
sudo /usr/share/thingsboard/bin/install/upgrade.sh --fromVersion=3.4.4

**Expected behavior**
A successfull update to 3.5 versionb

**Screenshots**

Starting ThingsBoard Upgrade from version 3.4.4 ...
Clearing cache to upgrade from version 3.4.4 to 3.5.0
Clearing cache [deviceProfiles]
Upgrading ThingsBoard from version 3.4.4 to 3.5.0 ...
Updating schema ...
Schema updated.
Installing SQL DataBase schema views and functions: schema-views-and-functions.sql
Unexpected error during ThingsBoard installation!
org.postgresql.util.PSQLException: ERROR: no existe la relación «asset_profile»
  Position: 1190
        at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676)
        at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366)
        at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356)
        at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:496)
        at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:413)
        at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:333)
        at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:319)
        at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:295)
        at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:290)
        at org.thingsboard.server.service.install.SqlAbstractDatabaseSchemaService.executeQueryFromFile(SqlAbstractDatabaseSchemaService.java:82)
        at org.thingsboard.server.service.install.SqlEntityDatabaseSchemaService.createOrUpdateViewsAndFunctions(SqlEntityDatabaseSchemaService.java:53)
        at org.thingsboard.server.install.ThingsboardInstallService.performInstall(ThingsboardInstallService.java:252)
        at org.thingsboard.server.ThingsboardInstallApplication.main(ThingsboardInstallApplication.java:49)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
        at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:467)
Unexpected error during ThingsBoard installation!
ThingsBoard upgrade failed!

**Additional context**
The current 3.4.4 version was running perfectly well, until last week where we detected that the dashboard did not display data before Oct 18th (6 days back) so we decided to update the platform. We confirmed that data is there but for any unknow reason is not displayed in dashboard. if we pickup shorter data history, data is displayed...

**Comment by trikimiki** [comment]
Hi!

> some data was not displayed on the dashboard, lets say display temp of the last 30 days, only showed 6 days, and nothing before

Can you check if any TTL value is present in the configuration and/or UI? If there are exactly 6 days, there could be a TTL set for 7 days.

> org.postgresql.util.PSQLException: ERROR: no existe la relación «asset_profile»

I suspect that either of those two is a root cause:

- you have installed a new version's package, that updated YML and CONF configs, changing the DB connection param
- there is an issue with your DB schema and table "asset_profile" is absent for some reason; it is highly unlikely, as in that case v3.4.4 would not start at all, but just in case you can check your tables against this schema: https://github.com/thingsboard/thingsboard/blob/v3.4.4/dao/src/main/resources/sql/schema-entities.sql

**Comment by tatuiot** [comment]
Hi, Finally I found the 6 days, was the DASHBOARD_MAX_DATAPOINTS_LIMIT set to 50.000 so I rised to 120K and extended to nearly 15 days the display.  so fixed.

And regarding the upgrade to 3.6 it must have been an error on my dev server, because, I did it in my prod server, clear the cache on the browser after upgrading , some dispay data were in the format day.hour.minutes and after that, works like a charm, thanks a lot. sorry for disturbing...
